src/Event/Http/FpmHandler.php

```diff
@@ -79,7 +84,7 @@ public function start(): void
         $this->fpm->setTimeout(null);
         $this->fpm->start(function ($type, $output): void {
             // Send any PHP-FPM log to CloudWatch
-            echo $output;
+            $this->logToCloudwatch($output);
```

Comment:
I think we shouldn't edit this part: these are the logs coming from the PHP application (for the most part at least).

Reply:
They weren't previously going to CW due to no line ending from my tests, hence adding this based on the comment that was already there above indicating it should have been. Happy to drop through if you don't feel this is appropriate :)

Reply:
Yes this is intentional: we are forwarding the PHP logs as-is. If we were to add a line return, we would end up with 2 line returns whenever a PHP logger logs for example.

And in any case I don't think we should add the JSON part on every line of PHP logs (too much of a change for users).